Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM fedora:25
```

Comment:
if we're sunsetting the site, why bother updating the container?  This will all be EOL shortly anyway.